Sources/Hyperspace/Request/Request.swift

```diff
@@ -65,8 +65,9 @@ public protocol Request {
     /// Attempts to parse the provided Data into the associated response model type for this request.
     ///
     /// - Parameter data: The raw Data retrieved from the network.
+    /// - Parameter serviceSuccess: The successful result of executing a Request using a NetworkService.
     /// - Returns: A result indicating the successful or failed transformation of the data into the associated response type.
-    func transformData(_ data: Data) -> Result<ResponseType, ErrorType>
+    func transformData(_ data: Data, serviceSuccess: NetworkServiceSuccess) -> Result<ResponseType, ErrorType>
```

Comment:
I think this is a good evolution of this function for now, but we may want to revisit this if we end up doing lots of breaking changes on the road to v3.0. I could see us possibly moving/copying the `data` property on `HTTP.Response` up to `NetworkServiceSuccess` and making it non-optional. That would make `NetworkServiceSuccess` feel a bit more useful and mirror the `error` property on `NetworkServiceFailure` better. Then our transformation function could just take a `NetworkServiceSuccess`, but then we would probably want to rename the function, breaking lots of things. 😄

Reply:
Ignore most of that. I hadn't pulled the latest changes so it looked like `NetworkServiceSuccess` didn't have the `data` property anymore. We're good!